src/storage/cdb/scripts/config.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This configuration is used by build and vendorizing scripts
+
+export PROJECT_NAME="cdb_rs"
+export TOOLCHAIN_VERSION="dev"
+export VENDORED_SOURCES_SUFFIX="thirdparty"
+export VENDORED_SOURCES_PACKER_ROLE="io-perf"
```

Comment:
leaking Twitter-specific concept such as packer and service role here

Reply:
I mean, i'm open to suggestions. Seems like kind of a hassle to have this stored separately.

One thing we could do to keep this out of master would be to put it in an orphan branch (like `gh-pages`). That way it's still "in the repo" but it's not part of the codebase.